sui_core/src/authority/authority_store.rs

```diff
@@ -834,10 +834,10 @@ impl<const ALL_OBJ_VER: bool, S: Eq + Serialize + for<'de> Deserialize<'de>>
         let (sequenced_to_write, schedule_to_write): (Vec<_>, Vec<_>) = ids
             .zip(versions.iter())
             .map(|(id, v)| {
-                let version = v.unwrap_or_else(SequenceNumber::new);
+                let version = v.unwrap_or_else(|| OBJECT_START_VERSION);
```

Comment:
Could you add comment here on why it must be OBJECT_START_VERSION?

Reply:
Fixed